content/en/docs/tasks/administer-cluster/kubeadm/kubeadm-upgrade.md

```diff
@@ -113,6 +113,8 @@ sudo yum list --showduplicates kubeadm --disableexcludes=kubernetes
 {{% /tab %}}
 {{< /tabs >}}
 
+If you don't see the version you expect to upgrade to, [verify if the Kubernetes package repositories are used.](/docs/tasks/administer-cluster/kubeadm/change-package-repository/#verifying-if-the-kubernetes-package-repositories-are-used)
```

Comment:
I'd suggest we put this a little bit earlier in this page, something like the following
as a precondition to check.

```suggestion
Please make sure you are using [the Kubernetes package repositories](/docs/tasks/administer-cluster/kubeadm/change-package-repository/#verifying-if-the-kubernetes-package-repositories-are-used)
when upgrading a cluster.
```

Reply:
Thanks for your input @tengqm ! The case is there's a similar reference to what you say [just above](https://kubernetes.io/docs/tasks/administer-cluster/kubeadm/kubeadm-upgrade/#changing-the-package-repository) the section where I want to include this comment. I was wondering if it won't be better to just rephrase that one, because for me (quite newby in Kubernetes) it wasn't straightforward connection I should change the package repository to be able to download other Kubernetes version. Thus I believe it's important to include the 'version' keyword in this context, nonetheless of where it would be

What do you think?